vllm/entrypoints/openai/api_server.py

```diff
@@ -162,27 +178,42 @@ async def show_available_models():
     return ModelList(data=model_cards)
 
 
-def create_logprobs(token_ids: List[int],
-                    id_logprobs: List[Dict[int, float]],
-                    initial_text_offset: int = 0) -> LogProbs:
+def create_logprobs(
+    token_ids: List[int],
+    top_logprobs: Optional[PromptLogprobs] = None,
```

Comment:
Using `PromptLogporbs` here is confusing since this function is not only for prompt log probs. Let's just use `List[Optional[Dict[int, float]]]` here for clarity?